precompiles/src/solidity/codec/xcm.rs

```diff
@@ -152,9 +154,11 @@ pub(crate) fn network_id_from_bytes(encoded_bytes: Vec<u8>) -> MayRevert<Option<
 				block_hash,
 			}))
 		}
+		/*
 		5 => Ok(Some(NetworkId::Westend)),
 		6 => Ok(Some(NetworkId::Rococo)),
 		7 => Ok(Some(NetworkId::Wococo)),
```

Comment:
Here, we can return an error
```suggestion
		7 => Err(RevertReason::custom("Wococo Network is no longer supported").into()),
```

Reply:
Addressed in https://github.com/polkadot-evm/frontier/pull/1614/commits/7f7d5877ac287de232b63299bdb3409b6d68af93